radis/io/hitran.py

```diff
@@ -94,6 +94,7 @@ def cast_to_int64_with_missing_values(dg, keys):
     """replace missing values of int64 columns with -1"""
     for c in keys:
         if dg.dtypes[c] != int64:
+            dg.loc[dg[c] == "  "] = -1  # replace empty cells by -1, e.g. HCN
```

Comment:
I think you're replacing for all columns while you should only be replacing the "C" one here. Right syntax is dg.loc[dg[C]=="  ", C] 

Or you can use replace : https://stackoverflow.com/a/52851249

Reply:
alright, done

Reply:
done  :)